test/functionalTest/cases/2015_notification_templates/notification_templates_payload_as_text.test.DISABLED

```diff
@@ -1,44 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yes, a textual payload is part of notification_templates_with_decoded_chars_in_notifications.test, step 01:

     "payload": "A string with double-quotes: %22${A1}%22"